codesign/analysis_options.yaml

```diff
@@ -25,3 +25,4 @@ linter:
   rules:
     prefer_single_quotes: true
     unawaited_futures: true
+    constant_identifier_names: false
```

Comment:
Remove, this is against the Flutter style guide.

Reply:
Sure, removed.